lib/Conversion/TosaToEmitC/TosaToEmitC.cpp

```diff
@@ -415,8 +442,9 @@ class RsqrtOpConversion : public OpConversionPattern<tosa::RsqrtOp> {
 };
 
 /// Convert `tosa.fully_connected` into an `emitc.call` operation.
-template <typename SrcOp, typename Adaptor = typename SrcOp::Adaptor>
-SmallVector<Value, 2>
+template <size_t operands, typename SrcOp,
```

Comment:
Why do we need to pass operands here in template? SmallVector can be resized and seems like it has to match Adaptor::getNumOperands().

Reply:
Good point, for some reason I had assumed we would need to specify the SmallVector size. Fixed, thanks!